### PR TITLE
Improve repo documentation

### PR DESCRIPTION
**Pull request by fzliu**
- Added CONTRIBUTING.md
- Updated Towhee logo
- Moved files under `docs` to root directory

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+The Towhee community maintains a list of [Good First Issues](https://github.com/towhee-io/towhee/contribute).
+
+### Sharing a new operator or pipeline
+
+Pipeline and operator contributions to our Towhee Hub are just as valued as artwork, code, and documentation contributions. If you have a new model, useful script, or an `x2vec` pipeline that you'd like to share with the Towhee community, [get in touch](towhee-team@zilliz.com) with us!
+
+### Submitting a new issue or feature request
+
+Please follow the templates we provide for submitting issues, bugs, and/or feature requests; try to avoid opening blank issues whenever possible.
```
Comment (shiyu22) [inline CONTRIBUTING.md:15]: It can add link [submitting issues, bugs, and/or feature requests](https://github.com/towhee-io/towhee/issues/new/choose)

**Comment by shiyu22** [comment]
One thing is that can you add an introduction about how to build towhee from source in README? For example:

```bash
git clone https://github.com/towhee-io/towhee.git
python setup.py install
```

File: CONTRIBUTING.md (hunk 1 above)
Comment (GuoRentong) [reply CONTRIBUTING.md]: please add another PR for these modifications

**Pull request merged**